javascript/src/test/components/ChannelMessages/MessageScrollable.test.js

```diff
@@ -240,13 +298,33 @@ describe("MessageScrollableView tests", () => {
             }
         }
         const {getByText} = render(<MessageScrollableView messages={[exampleMessage]}/>);
-        setTimeout(function (){
-            const bracketElement = getByText(/!channel/);
-            expect(bracketElement.href).toMatch(null);
-        }, 500)
+        const bracketElement = getByText(/@channel/);
+        expect(bracketElement.getAttribute("href")).toEqual(null);
+        
     });
 
     test("Date and time render correctly", () => {
+        useSWR.mockReturnValue({
+            data: []
+        });
+        const exampleMessage = {
+            "type": "message",
+            "subtype": "channel_join",
+            "ts": "1610833617.007900",
+            "user": "U017218J9B3",
+            "text": "<!channel> This is an announcement",
+            "channel": "section-6pm",
+            "user_profile": {
+                "real_name": "Test Person"
+            }
+        }
+        const {getByText} = render(<MessageScrollableView messages={[exampleMessage]}/>);
+        const date = getByText("2021-01-16 21:46:57");
```

Comment:
Quick change: As documentation for the future students for 156 can you put a comment here briefly describing why this time is in UTC? Props to everyone on your team for figuring this issue out. Just say something along the lines like:

The timestamps in this test are in UTC because Jest tests and the Github tests use UTC timezone

Reply:
I will add this!